src/cyto_ml/data/decollage.py

```diff
@@ -118,7 +118,9 @@ def output_dir(self) -> None:
             os.mkdir(self.output_directory)
 
     def do_decollage(self) -> None:
-        """Not very lovely single function that replaces the work of the script."""
+        """Not very lovely single function that replaces the work of the script.
+        See cyto_ml.pipeline.pipeline_decollage - has the same code in it
+        """
```

Comment:
I looked at this and thought it wasn't worth refactoring any further - the logic belongs in the pipeline task and this `FlowCamSession` class was a placeholder for it.

We could also just delete this version...